.github/workflows/release.yml

```diff
@@ -79,12 +79,24 @@ jobs:
           repository: containerd/containerd
           ref: ${{ github.ref }}
           path: src/github.com/containerd/containerd
+      - name: Checkout Microsoft/hcsshim
+        if: startsWith(matrix.os, 'windows')
+        uses: actions/checkout@v2
+        with:
+          repository: Microsoft/hcsshim
+          path: src/github.com/Microsoft/hcsshim
 
       - name: Install Linux dependencies
         if: startsWith(matrix.os, 'ubuntu')
         run: |
           sudo apt-get install -y btrfs-tools libseccomp-dev
-
+      - name: Prepare hcsshim source
+        if: startsWith(matrix.os, 'windows')
+        run: |
+          SHIM_COMMIT=$(grep Microsoft/hcsshim vendor.conf | awk '{print $2}')
```

Comment:
This is getting run via Powershell and failing: https://github.com/estesp/actions-playground/runs/728854650

I think that's the only issue here as I assume you can do `shell: bash` to get around this problem?

Reply:
Fixed.